app/controllers/links_controller.rb

```diff
@@ -0,0 +1,38 @@
+class LinksController < ApplicationController
+  before_action :set_link, only: [:destroy, :update]
```

Comment:
Very minor but is it worth keeping the ordering of actions consistent across the routes file and the controllers (in the routes file destroy is at the end, at least for these two)? I guess both in the `before_action` and the actual actions defined below?